src/translations/en.json

```diff
@@ -118,7 +118,10 @@
     "ImportCSV": "Import CSV",
     "CsvFileName": "CSV file name",
     "CSVFileComment": "# Supported Equipments Types : ",
-    "CSVFileCommentContingencyList": "# Use \"|\" as a separator if a contingency contains several equipments",
+    "CSVFileCommentContingencyList1": "# Use \"|\" as a separator if a contingency contains several equipments",
+    "CSVFileCommentContingencyList2": "# Line 1,ID_line1",
```

Comment:
Maybe you can remove "# Transformer 2,ID_transformer2"
And replace "# Line 1,ID_line1" by "# N-1 on line, ID_line1"

Reply:
ok